TombEngine/Game/Lara/lara_helpers.cpp

```diff
@@ -966,20 +966,22 @@ void HandlePlayerElevationChange(ItemInfo* item, CollisionInfo* coll)
 	{
 		if (CanStepUp(*item, *coll))
 		{
-			item->Animation.TargetState = LS_STEP_UP;
```

Comment:
Are we sure that manual setting of `TargetState` is not necessary here? `SetStateDispatch` does not set it.

Reply:
The new code below means it's no longer required. `GetStateDispatch` checks for `LS_STEP_UP`, and the dispatch occurs according to this result.